frontend/src/main.jsx

```diff
@@ -5,6 +5,7 @@ import ReactDOM from "react-dom/client";
 import App from "./App";
 import Contact from "./pages/ContactPage";
 import HomePage from "./pages/HomePage";
+import Conditions from "./pages/ConditionPage";
```

Comment:
It could be easier to understand if you named your import we the same word... like 
import ConditionPage from ...